lib/client.js

```diff
@@ -23,11 +26,11 @@ var defaultConnectOptions = {
   clean: true
 };
 
-var defaultId = function() {
+defaultId = function () {
   return 'mqttjs_' + Math.random().toString(16).substr(2, 8);
 };
```

Comment:
you can define it as a function, `function defaultId`, so it follows the same pattern of `nop` behind.